config/samples/numaplane.numaproj.io_v1alpha1_monovertexrollout.yaml

```diff
@@ -19,6 +19,7 @@ spec:
       scale:
         min: 2
         max: 3
+        lookbackSeconds: 60
```

Comment:
just kept this in here after my testing, since it's useful I guess to verify it doesn't get touched during all of the rescaling ?